Gruntfile.js

```diff
@@ -27,59 +27,149 @@ module.exports = function(grunt) {
         },
         // Less compilation
         less: {
-            production: {
+            stacks: {
                 files: {
-                    'docs/assets/css/stacks-documentation.css': 'docs/assets/less/stacks-documentation.less',
-                    'lib/css/stacks.css': 'lib/src/stacks.less',
+                    'dist/css/stacks.css': 'lib/css/stacks.less'
                 }
             },
-            partials: {
+            // note that the docs CSS includes the full Stacks
+            docs: {
+                files: {
+                    'docs/assets/css/stacks-documentation.css': 'docs/assets/less/stacks-documentation.less'
+                }
+            },
+            stacks_partials: {
                 // these files are immediately deleted because we don't distribute them directly, but we must ensure
                 // that they compile (e.g. if a static file accidentally accesses the dynamic options, the full bundle will compile
                 // fine, but stacks-static alone will blow up)
                 files: {
-                    'lib/css/stacks-static.css': 'lib/src/stacks-static.less',
-                    'lib/css/stacks-dynamic.css': 'lib/src/stacks-dynamic.less',
+                    'tmp/css/stacks-static.css': 'lib/css/stacks-static.less',
```

Comment:
Not 100% sure how this works here, but should this be `'dist/css/stacks-static.css'` to match the .gitignore above?

Reply:
No, these are compiled into `tmp/ ` and then immediately deleted again ([here](https://github.com/StackExchange/Stacks/blob/feature/javascript-bundle/Gruntfile.js#L160)) -- we don't ship compiled versions of these, they just exist so that Core can compile the two parts separately.

Compiling them right here (into the temp folder) only serves the purpose of making sure that they *do* compile separately.